servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -142,7 +148,6 @@ public boolean updateX509CertRecord(X509CertRecord certRecord) {
                             new AttributeUpdate(KEY_PREV_SERIAL).put(certRecord.getPrevSerial()),
                             new AttributeUpdate(KEY_PREV_IP).put(certRecord.getPrevIP()),
                             new AttributeUpdate(KEY_PREV_TIME).put(getLongFromDate(certRecord.getPrevTime())),
-                            new AttributeUpdate(KEY_CLIENT_CERT).put(certRecord.getClientCert()),
```

Comment:
I have removed the option to update client cert for a record (as in the JDBC connection).